packages/rancher-windows-gmsa/charts/templates/credentialspec.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.credential.enabled -}}
+apiVersion: windows.k8s.io/v1
+kind: GMSACredentialSpec
+metadata:
+  name: {{ lower .Values.credential.domainJoinConfig.machineAccountName }}  #This is an arbitrary name but it will be used as a reference

```

Comment:
nit:
```suggestion
  name: {{ .Values.credential.domainJoinConfig.machineAccountName | lower }}  #This is an arbitrary name but it will be used as a reference

```